### PR TITLE
#37 Refactored second screen code

### PR DESCRIPTION
**Pull request by NikitaSukhanov**
All issues, existing by the moment, are fixed.

### CODE REVIEW

**Comment by maik-nack** [comment]
Please, rename pull request

File: app/src/main/java/com/mithridat/nonoconverter/ui/result/ResultActivity.java [hunk 1]
```diff
@@ -51,59 +54,71 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+            }
+        };
+
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setTitle(R.string.title_dialog_home_return);
+        builder.setIcon(R.drawable.ic_info)
+                .setMessage(R.string.msg_home_return)
+                .setPositiveButton(R.string.action_return, listenerReturn)
+                .setNegativeButton(android.R.string.cancel, listenerCancel)
+                .create().show();
```
Comment (maik-nack) [inline app/src/main/java/com/mithridat/nonoconverter/ui/result/ResultActivity.java]: `.show()` should be on next line

**Review by PawelMelnik — APPROVED** [review]
Nice coding standard

File: app/src/main/java/com/mithridat/nonoconverter/ui/result/ResultActivity.java [hunk 2]
```diff
@@ -51,59 +54,71 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+            public void onClick(DialogInterface dialog, int id) {
+                dialog.cancel();
+            }
+        };
+
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setTitle(R.string.title_dialog_done)
+                .setIcon(R.drawable.ic_done)
+                .setPositiveButton(android.R.string.ok, listener)
+                .create().show();
```
Comment (maik-nack) [inline app/src/main/java/com/mithridat/nonoconverter/ui/result/ResultActivity.java]: `.show()` should be on next line

**Review by maik-nack — APPROVED** [review]
Nice

**Pull request merged**